### PR TITLE
[CSFS-156] Build a To-Do App with REACT

### PR DESCRIPTION
**Pull request by Marienoir**
## Task: `Build a To-Do App with REACT`
- `I Built a To-Do App with REACT`

### CODE REVIEW

**Comment by Ireti203** [comment]
Attach the screenshot of your work.

**Comment by Marienoir** [comment]
Screeenshots of the todo app

todo1
todo3
todo2

**Comment by Marienoir** [comment]
I have added screenshots of the app and also fixed codefactor issue.
@Ireti203 Please review. Thanks

File: submissions/maryOkosun/react/public/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
```
Comment (Ireti203) [inline submissions/maryOkosun/react/public/index.html:41]: Remove commented out code

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by Marienoir** [comment]
Added screenshots of the "xxx entries left"

**Comment by Marienoir** [comment]
@Ireti203  Please review my PR. Thanks

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**